From e09cc603d81f00061565083aa1ab7216e02a7f0a Mon Sep 17 00:00:00 2001
From: Jiri Pirko <jiri@resnulli.us>
Date: Thu, 3 Jan 2013 22:49:03 +0000
Subject: [PATCH 491/858] net: kill dev->master

Nobody uses this now. Remove it.

Signed-off-by: Jiri Pirko <jiri@resnulli.us>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 85464ef271a0f5496f404c6a2f2dfbf1d76e1a49)

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
---
 include/linux/netdevice.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index f5a7e27..ec0eef2 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -1172,10 +1172,6 @@ struct net_device {
        * avoid dirtying this cache line.
        */
 
- struct net_device *master; /* Pointer to master device of a group,
-       * which this device is member of.
-       */
-
  struct list_head upper_dev_list; /* List of upper devices */
 
  /* Interface address info used in eth_type_trans() */
-- 
1.8.3.1